src/app/checks/create/SettingsSection.tsx

```
@@ -17,7 +17,7 @@ const tabs = [
   { name: 'Sharing', icon: UsersIcon },
 ]
 export default function SettingsSection() {
-  const {} = useCreateCheckStore((state) => state)
+  const {} = useCheckStore((state) => state)
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)